docs/source/installation/versions.rst

```diff
@@ -26,21 +26,21 @@ The three versions of Manim
 
 There are currently three main versions of manim. They are as follows:
 
-- **ManimCE**: The community edition of manim. This is the version documented by this website, and is named `manim <https://pypi.org/project/manim/https://pypi.org/project/manim/>`_ on pip.
-- `ManimGL <https://github.com/3b1b/manim>`_: The current version of manim that is used by 3blue1brown. It supports OpenGL rendering and interactivity, and is named ``manimgl`` on pip. You can find documentation for it `here <https://3b1b.github.io/manim/index.html>`_.
```

Comment:
the comma separates two different things being talked about here, should be added back.

Reply:
Done